sdk/metricsadvisor/perf-tests/ai-metrics-advisor/test/rootCauses.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import {
+  PerfStressOptionDictionary,
+  executeParallel,
+  getEnvVar
+} from "@azure/test-utils-perfstress";
+import { MetricsAdvisorTest } from "./metricsAdvisor.spec";
+interface MetricsAdvisorTestOptions {
+  count: number;
+}
+
+export class RootCauseTest extends MetricsAdvisorTest<MetricsAdvisorTestOptions> {
+  detectionConfigId: string;
+  incidentId: string;
+  public options: PerfStressOptionDictionary<MetricsAdvisorTestOptions> = {
+    count: {
+      required: true,
+      description: "Number of Root Causes to be listed",
+      longName: "count",
+      defaultValue: 10
+    }
```

Comment:
Do we have at least 10 root causes, 10 anomalies, and 10 incidents associated with the alert ID used for these tests? If we try to list 10 root causes but have only 3 in the service, we could end up thinking the SDK is faster than it actually is. I wonder if it's possible and worth setting a maximum value for these parameters based on what we know about the alert used.

Reply:
I did find multiple anomalies and incidents for an example of alert. But not multiple root causes. Most of the examples I have come across only have a single root cause for an incident. I have asked Bowen on the channel about that one in particular.